lib/src/test/kotlin/net/dean/jraw/test/unit/GildingsTest.kt

```diff
@@ -43,25 +43,23 @@ class GildingsTest : Spek({
     }
 
 
-    describe("test new comment gildings, they should not be null") {
+    describe("comment gildings") {
 
         val ref = TestConfig.reddit.subreddit("iama").posts()
-            .limit(Paginator.RECOMMENDED_MAX_LIMIT)
             .sorting(SubredditSort.TOP)
-            .timePeriod(TimePeriod.ALL)
+            .timePeriod(TimePeriod.MONTH)
+            .limit(3)
             .build()
 
-        val subs = ref.next()
-        val submission = subs[0]
-
+        val submission = ref.next()[0]
         val comments = submission.toReference(TestConfig.reddit).comments()
 
         it("should have gildings") {
 
             val sequence = comments.walkTree().take(20)
             sequence.forEach {
 
-                if((it !is RootCommentNode)) {
+                if(it !is RootCommentNode) {
```

Comment:
Space between operator and punctuation

Reply:
I don't have this on my side: https://github.com/KirkBushman/JRAW/blob/master/lib/src/test/kotlin/net/dean/jraw/test/unit/GildingsTest.kt

Not on the IDE, nor on the browser

Reply:
Line 62?

```kotlin
if(it !is RootCommentNode) {
if (it !is RootCommentNode) {
```

Reply:
I thought you meant the exclamation point

Reply:
Sorry about that, I should have made it more clear what I was talking about.